clinical_refs/management/commands/seed_refs_diabetes.py

```
@@ -0,0 +1,39 @@
+    {
+        "title": "ADA Standards of Care 2025 – Glycemic Targets",
+        "source": "ADA",
+        "year": 2025,
+        "url": "https://example.org/ada/2025/glycemic-targets",
+        "topic": "diabetes",
+        "section": "glycemic-targets",
+        "lang": "en",
+        "tags": ["hba1c","targets","type2"],
+    },
+    {
+        "title": "ADA Standards of Care 2025 – Pharmacologic Therapy",
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)